### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jandvorak-sol60279**
corrected options for filestat -> path.stat()

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -602,13 +602,18 @@ path = ArtifactoryPath(
 # Get FileStat
 stat = path.stat()
 print(stat)
-print(stat.md5)
+print(ctime)
```
Comment (allburov) [inline README.md:605]: Where have we defined `ctime`? I don't see it above, probably it should be `stat.ctime`

Comment (jandvorak-sol60279) [reply README.md:605]: it is stat.ctime
sorry for that.

Comment (jandvorak-sol60279) [reply README.md]: should be fixed.

**Comment by allburov** [comment]
Děkuji!

**Pull request merged**